tools/topology/topology2/cavs/cavs-rt5682.conf

```diff
@@ -0,0 +1,240 @@
+<searchdir:cavs>
+<searchdir:include>
+<searchdir:include/common>
+<searchdir:include/components>
+<searchdir:include/dais>
+<searchdir:include/pipelines/cavs>
+<searchdir:cavs/platform/intel>
+
+<vendor-token.conf>
+<manifest.conf>
+<pdm_config.conf>
+<tokens.conf>
+<virtual.conf>
+<passthrough-playback.conf>
+<passthrough-capture.conf>
+<passthrough-be.conf>
+<data.conf>
+<pcm.conf>
+<pcm_caps.conf>
+<fe_dai.conf>
+<ssp.conf>
+<dmic.conf>
+<hda.conf>
+<hw_config.conf>
+<manifest.conf>
+<route.conf>
+<cavs/common_definitions.conf>
+<copier.conf>
+<pipeline.conf>
+<dai.conf>
+<host.conf>
+<dmic-default.conf>
+<hdmi-default.conf>
+
+Define {
+	MCLK 				24576000
+	NUM_DMICS			0
+	SPK_ID				7
```

Comment:
what is that?

Reply:
SPK_ID is the HDMI4_ID + 1, NUM_DMICS and MCLK are the default values can be overwritten.